syntax = "proto3";

package protocols.behavior;

import "protocols/behavior/motion.proto";
import "protocols/common/peripheral_actuation.proto";

message Planning {
  DiscretizedPath discretized_path = 1;
  GoToPoint go_to_point = 2;
  RotateInPoint rotate_in_point = 3;
  RotateOnSelf rotate_on_self = 4;
  common.PeripheralActuation peripheral_actuation = 5;
}
